.github/workflows/release.yml

```diff
@@ -0,0 +1,348 @@
+name: Release (neo-cli)
+
+# Trigger the workflow on a release event when a new release is published
+on:
+  release:
+    types: [published]
```

Comment:
So how about the manually-triggered builds, don't we need them?

Reply:
What are you talking about this is automatic on a `publish` release